app.js

```diff
@@ -0,0 +1,18 @@
+const orderUrl = ['https://testnet.binancefuture.com/fapi/v1/depth?symbol=BTCUSDT', 'Orders'];
+const tradesUrl = ['https://testnet.binancefuture.com/fapi/v1/depth?symbol=BTCUSDT', 'Trades'];

```

Comment:
What happens if I want to use another symbol ?

Reply:
I have refactored the code so that a symbol is passed to the function to get Data. Expect on next PR